web/pandas/_templates/layout.html

```diff
@@ -20,41 +20,44 @@
             <link rel="stylesheet"
                   href="{{ base_url }}{{ stylesheet }}">
         {% endfor %}
+        <script src="https://kit.fontawesome.com/79e5369384.js" crossorigin="anonymous"></script>
```

Comment:
My understanding is that loading stylesheets and JS should go at the bottom of the page, to not slow down the loading of the page content. Is that accurate?

Reply:
I followed fontawesome instructions, I don't know if there is a reason to have it in the header, or it doesn't matter, but it's where they ask to add it.

Not an expert, but I'd say browsers load those asynchronously. I assumed that they were placed at the end to have the dom loaded when the code is executed. But my knowledge about web is outdated 10 years, so don't really know.

Thanks for catching the broken link, just fixed them.